napari_aicssegmentation/_tests/util/lazy_test.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from napari_aicssegmentation.util.lazy import lazy_property
+
+
+class TestLazy:
+    def test_lazy_property(self):
+        class TestObject:
+            @lazy_property
+            def property1(self):
+                return object()
+
+        test_obj = TestObject()
+        obj_id = id(test_obj.property1)
+        for i in range(10):
+            assert obj_id == id(test_obj.property1)
```

Comment:
Can you tell me how this test works? Why is the for loop necessary?

Reply:
I'm testing lazy initialization on a property, meaning that the first access to the property should give me a new object instance, but any subsequent access needs to give me the same instance that was created earlier (not a new one every time). So I'm accessing the property several times and making sure the object's ID is always the same. Loop is probably overkill - I could have just done 2 asserts in a row, but why do 2 when you can do 10 :)

Reply:
Haha ok thanks